controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -527,7 +527,6 @@ bool master_thread::autoconfig_wsc_add_m2(std::shared_ptr<ieee1905_1::tlvWscM1>
         return false;
     }
 
-    int bss_type  = tlvWscM1->vendor_extensions_attr().subelement_value;
```

Comment:
This makes me realize that our definition of the vendor extension in M1 is wrong. M1 has a vendor extension element to contain the WSC version (the interpretation of the Version attribute was so badly specified in the first version of WSC that it was not possible to put "version 2" in it, so they had to add a vendor-specific element to carry the "version 2" information). It happens to work correctly because the length of the version2 element is the same as the length of the MultiAP vendor extension in M2, and because the tlvf doesn't check the values. But to be really correct, we need to have separate definitions.

Reply:
I assume this is handled in #246 ?

Reply:
rebased on top of #246 and removed the second obsolete commit